rio/src/screen/constants.rs

```diff
@@ -24,3 +24,9 @@ pub const DEADZONE_END_Y: f64 = -2.0;
 
 #[cfg(target_os = "macos")]
 pub const DEADZONE_START_X: f64 = 80.;
+
+#[cfg(all(
+    any(feature = "wayland", feature = "x11"),
+    not(any(target_os = "macos", windows))
+))]
+pub const APPLICATION_ID: &str = "rio";
```

Comment:
Does Rio with uppercase `R` works as well?

Reply:
Asking because first letter uppercased is default for other platforms.

and alacritty also does have uppercased for wayland/x11 https://github.com/alacritty/alacritty/blob/master/alacritty/src/config/window.rs#L19C11-L19C23

Reply:
Waybar looks for desktop files based on the app_id (https://github.com/Alexays/Waybar/blob/3ecd4030e3318863d2bab087d01c9921624f0269/src/modules/wlr/taskbar.cpp#L66C1-L90C1), so we'd probably have to rename the desktop file to match.

Reply:
Asking because Rio with uppercased `R` is the default for other platform.

and alacritty also sets with first letter uppercased for x11/wayland https://github.com/alacritty/alacritty/blob/master/alacritty/src/config/window.rs#L19

Reply:
https://github.com/alacritty/alacritty/blob/2101d5ac37d65114781a84b889d96eafe768860a/extra/linux/Alacritty.desktop#L4

Reply:
> and alacritty also sets with first letter uppercased for x11/wayland https://github.com/alacritty/alacritty/blob/master/alacritty/src/config/window.rs#L19

Alacritty might be an outlier:

![obraz](https://github.com/raphamorim/rio/assets/5671049/cfdc0fb0-1e5d-41cf-8f30-f1117e66845a)

Reply:
> Does Rio with uppercase `R` works as well?

Yeah, Waybar seems to pick it up no matter the casing.

Reply:
hmmm, let's keep lowercased then to match with wezterm and kitty

Reply:
Did you check if for multiwindow it works without any inconsistency?

Reply:
> Did you check if for multiwindow it works without any inconsistency?

Yup, all windows have the app_id, and all get an icon in Waybar.